src/translations/en.json

```diff
@@ -8,5 +8,189 @@
     "log_out": "Log out",
     "mailbox": "Mailbox",
     "shopping_list": "Shopping list"
+  },
+  "state": {
+    "default": {
+      "unknown": "Unknown",
+      "unavailable": "Unavailable"
+    },
+    "alarm_control_panel": {
+      "armed": "Armed",
+      "disarmed": "Disarmed",
+      "armed_home": "Armed home",
+      "armed_away": "Armed away",
+      "armed_night": "Armed night",
+      "pending": "Pending",
+      "arming": "Arming",
+      "disarming": "Disarming",
+      "triggered": "Triggered"
+    },
+    "automation": {
+      "off": "Off",
```

Comment:
[%key:state::switch::off%]

Reply:
We need to decide which file is the "fallback" one.
If this one is the fallback, then we can't have keys here.
Or we can use keys here and use translations/en.json as fallback.

Reply:
Good points. I see three options we have here.

I don't like using translations/en.json myself, for a few reasons. First of all, it means it's going to complicate local development, since translations/en.json is only fetched from Lokalise. We could build against src/translations/en.json in dev mode, but it means we'd be getting the raw key references. It also means that after merging new translation keys in src, we'd be commiting the repo into a state with invalid key lookups, until the next round-trip to Lokalise.

I'm personally leaning towards src/translations/en.json. I'm not too worried about including duplicates in the base translation myself. I think that feature is going to be much more valuable for translators, especially since the app is natively written in English, we don't have to worry as much about wanting to change how "Off" looks everywhere. The nested placeholder example of cancel/proceed seems more useful, but I think in most of those cases we'd probably have variable placeholders anyway.